RELEASES.md

```diff
@@ -1,3 +1,132 @@
+- [Add assert_unsafe_precondition to unchecked_{add,sub,neg,mul,shl,shr} methods](https://github.com/rust-lang/rust/pull/121571/)
+- [Update `c_char` on AIX to use the correct type](https://github.com/rust-lang/rust/pull/122986/)
+- [Remove libc from MSVC targets](https://github.com/rust-lang/rust/pull/124050/)
+- [Update libc to 0.2.155](https://github.com/rust-lang/rust/pull/124560/)
+- [`offset_of!` no longer returns a temporary](https://github.com/rust-lang/rust/pull/124484/)
+- [Hangle sigma in `str.to_lowercase` correctly](https://github.com/rust-lang/rust/pull/124773/)
+- [Raise `DEFAULT_MIN_STACK_SIZE` to at least 64KiB](https://github.com/rust-lang/rust/pull/126059/)
+- [`std::env::{set_var, remove_var}` can no longer be converted to safe function pointers and no longer implement the `Fn` family of traits](https://github.com/rust-lang/rust/pull/124636)
+
+
+<a id="1.80-Stabilized-APIs"></a>
+
+Stabilized APIs
+---------------
+- [Correct the const stabilization of `last_chunk` for slices](https://github.com/rust-lang/rust/pull/124836/)
+- [Implement Default for `Arc<str>`/`Rc<str>`/`Arc<[T]>`/`Rc<[T]>`/`Arc<CStr>`](https://github.com/rust-lang/rust/pull/124640/)
+- [Implement `IntoIterator for Box<[T]>` + edition 2024-specific lints](https://github.com/rust-lang/rust/pull/124097/)
+- [Implement `FromIterator for Box<str>`](https://github.com/rust-lang/rust/pull/99969/)
+- [Stabilize `LazyCell` and `LazyLock`](https://github.com/rust-lang/rust/pull/121377/)
+- [Stabilize `div_duration`](https://github.com/rust-lang/rust/pull/124667/)
+- [Stablize `const_binary_heap_constructor`](https://github.com/rust-lang/rust/pull/125211/)
+- [Stabilize Option::take_if](https://github.com/rust-lang/rust/pull/126089/)
+- [Stabilize `seek_seek_relative`](https://github.com/rust-lang/rust/pull/123817/)
+- [Stabilize `binary_heap_as_slice`](https://github.com/rust-lang/rust/pull/124012/)
+- [Stabilize `non_null_convenience`](https://github.com/rust-lang/rust/pull/124498/)
+- [Stabilize `split_at_checked`](https://github.com/rust-lang/rust/pull/124678/)
+- [Stabilize `byte_slice_trim_ascii` for `&[u8]`/`&str`](https://github.com/rust-lang/rust/pull/124928/)
+- [Stabilise `IpvNAddr::{BITS, to_bits, from_bits}` (`ip_bits`)](https://github.com/rust-lang/rust/pull/125551/)
+- [Stabilize `slice_flatten`](https://github.com/rust-lang/rust/pull/125561/)
+
+
+<a id="1.80-Cargo"></a>
+
+Cargo
+-----
+**relnotes**
+- [Stabilize `-Zcheck-cfg` as always enabled](https://github.com/rust-lang/cargo/pull/13571/)
+- [Warn, rather than fail publish, if a target is excluded](https://github.com/rust-lang/cargo/pull/13713/)
+- [Add special `check-cfg` lint config for the `unexpected_cfgs` lint](https://github.com/rust-lang/cargo/pull/13913/)
+- [Stabilize `cargo update --precise <yanked>`](https://github.com/rust-lang/cargo/pull/13974/)
+- [Build only the specified artifact library when multiple types are available](https://github.com/rust-lang/cargo/pull/13842/)
```

Comment:
This is a nightly-only change, which I don't think are normally included in the relnotes here.

```suggestion
```

Reply:
Thanks :-) I was finding it a bit hard to tell what cargo stuff was stable vs unstable just from the PRs

Reply:
Yea, it can be hard without context.

Cargo's [CHANGELOG.md](https://github.com/rust-lang/cargo/blob/master/CHANGELOG.md#cargo-180-2024-07-25) can help since it summarizes each PR, and categorizes the changes, and nightly-only stuff is in a separate section.

Reply:
Oh! That does seem very handy, thanks